src/server/req_runjob.c

```diff
@@ -1310,6 +1334,16 @@ post_sendmom(struct work_task *pwt)
 				        "send of job to %s failed error = %d",
 				        jobp->ji_qs.ji_destin, pbs_errno);
 			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO, jobp->ji_qs.ji_jobid, log_buffer);
+			snprintf(log_buffer, sizeof(hook_buf),
+					"Not Running: PBS Error: %s", pbse_to_txt(PBSE_MOMREJECT));
```

Comment:
I'm not necessarily sure this is a new comment.  If the runjob request is rejected, the comment is in the form as above.  So if the mom rejected the request and that error code made it back to the scheduler, the scheduler would print the same thing.  The question is if this would happen without async runjob.  Nithin, could you test this with throughput mode off?  Basically turn off throughput mode and rerun your test.  If the comment is the same with and without throughput mode, you won't need an EDD.